test/Interop/Cxx/reference/Inputs/reference.h

```diff
@@ -0,0 +1,13 @@
+#ifndef TEST_INTEROP_CXX_REFERENCE_INPUTS_REFERENCE_H
```

Comment:
Please also add a module interface printing test. We should ensure that we don't import references as implicitly unwrapped optionals.

Reply:
Done.